Documentation/release-notes/4645.md

```diff
@@ -0,0 +1,5 @@
+#### Application behavior on device and emulator
+
+  * [GitHub 4596](https://github.com/xamarin/xamarin-android/issues/4596):
+    Ensure that Java-to-managed type mappings lookup the correct non-generic
+    binding type, avoiding a *System.MemberAccessException*.
```

Comment:
Looks good. As a heads-up, I'll plan to adjust this to the following for the final aggregation of the release notes:

```
#### Application behavior on device and emulator

  * [GitHub 4596](https://github.com/xamarin/xamarin-android/issues/4596):
    Starting in Xamarin.Android 10.3 Preview, *System.MemberAccessException*
    could cause apps to abort when using certain APIs involving generic types.
```

I've got in the habit of using the "Starting in..." phrase to let users know the scope of the fix for regressions and to help me see which items can removed from the final GA release notes. The other adjustment is to move the user-visible message text to appear near the beginning, just after that "Starting in..." phrase, because that text is what a user would be most likely to look for to recognize the issue if they were hitting it.

Further suggestions on the note are welcome, either as comments on this PR or as changes in the committed note, but no need to change the committed note otherwise, unless you want to. Thanks!